src/pages/index.jsx

```diff
@@ -287,6 +304,41 @@ let caseStudies = [
   }
 ];
 
+let templates = [
+  {
+    href: 'https://apes.io',
+    screenshot: imgAstro,
+  },
+  {
+    href: 'https://rivet.gg/learn/godot/tutorials/crash-course',
+    screenshot: imgBomber,
+  },
+  {
+    href: 'https://rivet.gg/learn/html5/tutorials/tanks-canvas-socketio',
+    screenshot: imgTanks,
+  },
+  {
+    href: 'https://rivet.gg/learn/unity/tutorials/fishnet/crash-course',
+    screenshot: imgUnityTanks,
+  },
+  {
+    href: 'https://github.com/rivet-gg/examples/tree/main/html5/colyseus',
+    screenshot: imgCol,
+  },
+  {
+    href: 'https://github.com/rivet-gg/examples/tree/main/html5/webrtc',
+    screenshot: imgWeb,
+  },
+  {
+    href: 'https://github.com/rivet-gg/examples/tree/main/rust/cli-websocket',
+    screenshot: imgCli,
+  },
+  {
+    href: 'https://github.com/rivet-gg/examples/tree/main/c/coredumpo',
```

Comment:
typo, should be https://github.com/rivet-gg/examples/tree/main/c/coredumpohttps://github.com/rivet-gg/examples/tree/main/c/coredump